app/models/user.rb

```diff
@@ -127,9 +127,10 @@ def check_location_updated
   end
 
   def send_welcome_email
-    if self.roles_name.include? "rider"
-      UserMailer.welcome_email_rider(self).deliver_later
-    elsif self.roles_name.include? "driver"
+    return if has_sms_name?
+    if self.roles_name.include? 'voter'
```

Comment:
I didn't know about the `#has_role? :driver, :any` type method when I wrote this, I just inspected the object to find something that worked.  If you want, you could use that cleaner way.